core/src/main/scala/anorm/Column.scala

```diff
@@ -713,10 +713,11 @@ sealed trait JavaTimeColumn {
   implicit val columnToInstant: Column[Instant] = nonNull { (value, meta) =>
     val MetaDataItem(qualified, _, _) = meta
     value match {
+      case ts: java.sql.Timestamp => Right(ts.toInstant)
       case date: java.util.Date => Right(Instant ofEpochMilli date.getTime)
       case time: Long => Right(Instant ofEpochMilli time)
-      case TimestampWrapper1(ts) => Ts(ts)(Instant ofEpochMilli _.getTime)
```

Comment:
Why not `Ts(ts)(_.toInstant)` to keep it consistent?

Reply:
this is a good point, I'll correct this